src/renderer/components/cluster-view/EthPipeRenderer.tsx

```diff
@@ -12,7 +12,7 @@ import {
     getEpochNormalizedTotalOps,
     getLinkSaturation,
     getShowLinkSaturation,
-    getTotalOpsForGraph,
+    getTotalOps,
 } from '../../../data/store/selectors/linkSaturation.selectors';
 import { getFocusPipe, getSelectedPipesIds } from '../../../data/store/selectors/pipeSelection.selectors';
 import { getHighContrastState } from '../../../data/store/selectors/uiState.selectors';
```

Comment:
`calculateLinkSaturationMetrics` if accessed directly shoudl not be in store or shoudl not be accessed directly

Reply:
I couldn't think of where to put it... open to suggestions 😕